### PR TITLE
[Merged by Bors] - feat(Topology/Instances/EReal): add continuity of multiplication

### PR DESCRIPTION
**Pull request by pitmonticone**
This PR adds the continuity of multiplication. 

Co-authored-by: @D-Thomine 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Instances/EReal.lean [hunk 1]
```diff
@@ -245,4 +245,122 @@ instance : ContinuousNeg EReal := ⟨negOrderIso.continuous⟩
...
+    (h : ContinuousAt (fun p : EReal × EReal ↦ p.1 * p.2) (a, b)) :
+    ContinuousAt (fun p : EReal × EReal ↦ p.1 * p.2) (b, a) := by
+  have : (fun p : EReal × EReal ↦ p.1 * p.2) = (fun p : EReal × EReal ↦ p.1 * p.2)
+      ∘ Prod.swap := by
+    ext p
+    simp only [Function.comp_apply, Prod.fst_swap, Prod.snd_swap]
+    exact mul_comm p.1 p.2
+  rw [this]
+  apply ContinuousAt.comp _ (Continuous.continuousAt continuous_swap)
+  simp [h]
```
Comment (sgouezel) [inline Mathlib/Topology/Instances/EReal.lean:264]:
```suggestion
  convert h.comp continuous_swap.continuousAt (x := (b, a))
  simp [mul_comm]
```
Just a little golf. Not really important, but just to show some tricks.

**Review by sgouezel — COMMENTED** [review]
Looks very good, thanks!
bors d+

File: Mathlib/Topology/Instances/EReal.lean (hunk 1 above)
Comment (pitmonticone) [reply Mathlib/Topology/Instances/EReal.lean]: Done. Thanks.

**Pull request closed without merging**